src/modules/login/ResetPassword.tsx

```diff
@@ -15,28 +15,41 @@ import Toast from "../../utils/Toast";
 import { useHistory, useParams } from "react-router";
 import { hasNoValue } from "../../components/inputs/inputHelpers";
 import Error from "../../components/Error";
+import PasswordChecklist from "react-password-checklist";
```

Comment:
is this a new library?

Reply:
I just committed new changes. For some reason I can't attach a screenshot of it to the PR description:
![Screen Shot 2021-06-17 at 12 38 55 PM](https://user-images.githubusercontent.com/68650343/122373147-ed9df580-cf69-11eb-93cd-f772d4b7b7d8.png)